index.html

```diff
@@ -56,24 +56,28 @@ <h1 id="header">Basics: Beat That! 🚀</h1>
       <input id="input-field" />
       <button id="submit-button">Submit</button>
       <p>Output:</p>
-      <div id="output-div"></div>
+      <div id="output-div">
```

Comment:
@jeremylim91 could you help to rebase on top of `main` to avoid these changes? thank you!